ControlScheme.json

```diff
@@ -41,6 +41,13 @@
     },
     "Right Stick Y (Xbox)"
   ],
+  "defenseStateControl": [
+    0,
+    {
+      "Button": 4
+    },
+    "Y (Xbox)"
```

Comment:
Same button as resetDrive

Reply:
reset drive doesnt exist in the current codebase...